graylog2-server/src/test/java/org/graylog2/inputs/codecs/GelfDecoderTest.java

```diff
@@ -392,4 +401,29 @@ public void decodeSucceedsWithValidTimestampAsStringIssue4027() {
         assertThat(message).isNotNull();
         assertThat(message.getTimestamp()).isEqualTo(DateTime.parse("2017-07-21T14:23:00.661Z"));
     }
+
+    @Test
+    public void decodeFailsWhenMultipleMessages() {
+        final String json = "{\"short_message\":\"Bulk message 1\", \"host\":\"example.org\", \"facility\":\"test\", \"_foo\":\"bar\"}\n" +
+                "\n" +
+                "{\"short_message\":\"Bulk message 2\", \"host\":\"example.org\", \"facility\":\"test\", \"_foo\":\"bar\"}\n" +
+                "\n" +
+                "{\"short_message\":\"Bulk message 3\", \"host\":\"example.org\", \"facility\":\"test\", \"_foo\":\"bar\"}\n" +
+                "\n" +
+                "{\"short_message\":\"Bulk message 4\", \"host\":\"example.org\", \"facility\":\"test\", \"_foo\":\"bar\"}\n" +
+                "\n" +
+                "{\"short_message\":\"Bulk message 5\", \"host\":\"example.org\", \"facility\":\"test\", \"_foo\":\"bar\"}";
+
```

Comment:
```suggestion
final String json = """
                {"short_message":"Bulk message 1", "host":"example.org", "facility":"test", "_foo":"bar"}
                
                {"short_message":"Bulk message 2", "host":"example.org", "facility":"test", "_foo":"bar"}
                
                {"short_message":"Bulk message 3", "host":"example.org", "facility":"test", "_foo":"bar"}
                
                {"short_message":"Bulk message 4", "host":"example.org", "facility":"test", "_foo":"bar"}
                
                {"short_message":"Bulk message 5", "host":"example.org", "facility":"test", "_foo":"bar"}
                """;

```
You can make use of a multi-line string, which makes it a lot more readable IMO :slightly_smiling_face: